### ISSUE TITLE
multisig: display status of number of signatures/signees wrong when >1 input is used

### ISSUE THREAD

**Issue by accumulator** [spesmilo/electrum#8194]
Transaction status string incorrectly calculates number of signatures and number of signees when more than one input is used.

E.g. when using a 1 of 3 wallet, and initiating a payment on the first wallet using 2 inputs, the transaction status shows as
'Partially signed (2/4)'. The partially signed transaction can be imported and signed by the second wallet fine, after the second signature the transaction is complete.

**Comment by SomberNight** [comment]
> E.g. when using a 1 of 3 wallet, and initiating a payment on the first wallet using 2 inputs, the transaction status shows as
'Partially signed (2/4)'. The partially signed transaction can be imported and signed by the second wallet fine, after the second signature the transaction is complete.

Are you sure this was a 1of3 wallet? Even from your description it sounds like it is a 2of3? Why did you import it in the second wallet otherwise?
For 2 inputs and a 2of3 wallet, 'Partially signed (2/4)' is correct after the first wallet signs.
https://github.com/spesmilo/electrum/blob/5eb7bcebef9c36300db9b2edd6e1e9e9c695dea8/electrum/transaction.py#L1997-L2006
At that time we have `2*1` sigs, and will need `2*2` sigs altogether.

**Comment by accumulator** [comment]
Oops, yes a 2of3 wallet of course

**Comment by accumulator** [comment]
I guess this issue then becomes a debate between the technical interpretation (number of signatures on the inputs) vs the functional interpretation (number of signers).

I'd say most users would expect the latter. The technical aspects should IMO be visible in a technical details (section of) the transaction details screen

**Comment by SomberNight** [comment]
Note that you are assuming the inputs are homogeneous. You could have a tx with a combination of some singlesig inputs, some 2of2, some 3of4, etc. Of course in practice this rarely happens.

**Comment by accumulator** [comment]
In these heterogenous cases you probably want a more detailed view of each of the inputs participating in a transaction, to see which inputs are part of the wallet and which are external. 

Even then, I don't think you can know how many signers (or signatures) are needed for the external inputs

**Comment by accumulator** [comment]
I changed the title to make this more of a discussion instead of a bug.

It might be more clear to users if we could show (maybe in addition to the existing x/y signatures) e.g. something like 'Signatures from 1 additional co-signer are needed'

**Comment by ecdsa** [comment]
indeed, it might be more intuitive to display the number of signers who signed
